src/device_stm32h5.c

```diff
@@ -12,6 +12,8 @@
 #define FLASH_NSCCR (FLASH_BASE + 0x30)
 #define FLASH_OPTSR_CUR (FLASH_BASE + 0x50)
 #define FLASH_OPTSR_PRG (FLASH_BASE + 0x54)
+#define SCB (0xE000E000UL +  0x0D00UL)
+#define SCB_VTOR  (SCB + 0x008)
```

Comment:
This is ARM Cortex-M, it is not H5 specific, we could rely on CMSIS or to keep the baremetal approach just undef SCB and VTOR and define a pointer and a simplified struct to use SCB->VTOR.
This could go in device.h, it won't harm non-ARM devices if we undef and define our own

Reply:
@scaprile this code, like our ethernet drivers, does not pull CMSIS